src/bit_string.rs

```diff
@@ -12,6 +12,7 @@ pub struct BinaryTypeSpecifier<T> {
 }
 
 impl<T> BinaryTypeSpecifier<T> {
+    #[allow(clippy::too_many_lines)]
```

Comment:
Let's not enable this lint globally, I don't think it is useful

Reply:
I agree that it is annoying. I'll add it to the global allow because it comes as part of clippy::pedantic which is generally useful.